src/test/java/newRacingCar/domain/CarTest.java

```diff
@@ -0,0 +1,53 @@
+package newRacingCar.domain;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatExceptionOfType;
+
+public class CarTest {
+
+    private Car car;
+
+    @BeforeEach
+    void 설정() {
+        car = new Car(new Name("min"));
+    }
+
+    @Test
+    void 자동차_이름_길이_테스트() {
+        assertThatExceptionOfType(IllegalArgumentException.class)
+                .isThrownBy(() -> {
+                   Car car = new Car(new Name("minchoi"));
+                });
+    }
+
+    @Test
+    void 자동차_이동_테스트() {
+        car.move(new MoveStrategy() {
+            @Override
+            public boolean move() {
+                return true;
+            }
+        });
+        assertThat(car.getPosition()).isEqualTo(new Position(1));
+    }
+
+    @Test
+    void 자동차_멈춤_테스트() {
+        car.move(new MoveStrategy() {
+            @Override
+            public boolean move() {
+                return false;
+            }
+        });
```

Comment:
```suggestion
        car.move(() -> false);
```
요렇게 간단하게 쓸 수도 있습니다 :)